web/src/beta/components/RadioGroup/RadioBox/index.tsx

```diff
@@ -7,23 +7,31 @@ export type Props = {
   selected?: boolean;
   value: string;
   onClick?: (value: string) => void;
+  label?: string;
 };
 
-const RadioBox: React.FC<Props> = ({ inactive, selected, value, onClick }: Props) => {
+const RadioBox: React.FC<Props> = ({ inactive, selected, value, label, onClick }: Props) => {
   const [isChecked, setIsChecked] = useState(selected ?? false);
 
-  const handleRadioClick = useCallback(() => {
-    setIsChecked(!isChecked);
-    if (onClick) onClick(value);
-  }, [isChecked, onClick, value]);
+  const handleRadioClick = useCallback(
+    (value: string) => {
+      setIsChecked(!isChecked);
+      if (onClick) onClick(value);
```

Comment:
Just doing `onClick?.(value)` is enough. If there is an onClick, it'll be used, and if not it of course won't be used 👍🏻

Reply:
done